src/bindings/python/flux/job/executor.py

```diff
@@ -334,7 +334,10 @@ def result(self, *args, **kwargs):  # pylint: disable=arguments-differ
     result.__doc__ = concurrent.futures.Future.result.__doc__
 
     def set_exception(self, exception):
-        """When setting an exception on the future, set the jobid if it hasn't
+        """
+        Set an exception.
```

Comment:
Are the changes to this file necessary? Also, they don't seem related to the rest of the PR.

Reply:
This was probably me mindlessly changing the comment style to one I like... sorry! I can change it back.

Reply:
It's just docstrings, but I would probably change it back, yeah. If you were going to make the change, it would probably go in another commit, but I'm not 100% certain about that.